### PR TITLE
Inprogress

### PR DESCRIPTION
**Pull request by isabellecochrane**
an empty homepage with a working anchor to the portfolio page

### CODE REVIEW

**Comment by isabellecochrane** [comment]
please ignore the portfolio.html

File: portfolionav.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	        <a href=portfolionav.html>Portfolio</a>
+	    </li>
+	    <li>
+	        <a href=”/about”>About Me</a>
+	    </li>
+	    <li>
+	        <a href=”/contact”>Contact</a>
+	    </li>
+	</div>
+	  </ul>
```
Comment (dotNATE) [inline portfolionav.html:24]: ul closing tag and div closing tag wrong way round

File: portfolio.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+		display:flex;
+		display:inline;
+		align-items:center;
+		padding:0px 20px 0px 0px;
+}
+
+a {
+	text-decoration:none;
+	border-bottom:1px solid #FFE4E1;
+	color:FFE4E1;
```
Comment (dotNATE) [inline portfolio.css:49]: no # before color code

File: portfolio.css [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+	display:flex;
+	justify-content:space-between;
+	background-color:#87CEFA;
+}
+
+.name {
+	display:flex;
+	align-items:center;
+	margin:10px;
+	color:FFE4E1;
```
Comment (dotNATE) [inline portfolio.css:19]: no # before color code

**Review by dotNATE — COMMENTED** [review]
nesting on portfolionav.html is tough to read but otherwise pretty clear

File: portfolionav.html (hunk 1 above)
Comment (SobiaF) [reply portfolionav.html:24]: fix the  <a href=”/contact”>Contact</a> part?

**Pull request merged**